_parts/part03.md

```diff
@@ -209,6 +211,13 @@ In both base-2 and base-10, we can think of counting as starting with the number
 
 Given that you know how to add numbers in binary, this section won't be too long since you already know how to add 1 to any value. But, there are a few worthwile takeaways. When we count in decimal, we take the number we currently have and keep attempting to increment (add one) to the least significant digit position. In some cases this will work: $$8\rightarrow9, 95\rightarrow96,245\rightarrow246$$. However, in some instances, this won't work. For example in $$49$$, we can't simply increment the 9 to a 10 since 10 requires two digit positions and we only have 1 to work with. So, we roll over the 9 to a 0 and add a 1 to the next digit position. That is we roll the 9 in 49 to yield 40 and then try increment the next digit position from a 4 to a 5 which yields us the result of 50. Let's briefly think about why we carry over one. This might be obvious to some and less clear to others. If we have a 9 in the current digit position and we want to get to the next number, we have an issue with adding 1 to the number. However, we can accomplish the same result by simultaneously adding ten to the number and subtracting 9. Formally put, for some $$X$$, $$X+1=X+10-9$$. And that is exactly what we are doing when we count, when we roll the 9 to a 0, we are subtracting 9 from $$X$$, and when we are adding a 1 to the next digit position, that is synonymous with adding 10 to X. 
 
+| Adding 1 to 9 | | |
+|:--:|:--:|:--:|
+| | 0 | 9 |
+| + | 1 | 0 |
+| - | 0 | 9 |
+| **=** | **1** | **0** |
+
```

Comment:
Okay it took me a little bit to understand what was going on here. Do you think it would be more useful to add a line above the table along the lines of "Showing how incrementing a number can be accomplished by first adding ten to the base number followed by subtracting 9"